fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableClauseAnalyzer.java

```diff
@@ -344,7 +344,15 @@ public Void visitModifyTablePropertiesClause(ModifyTablePropertiesClause clause,
             } catch (DateTimeParseException e) {
                 ErrorReport.reportSemanticException(ErrorCode.ERR_COMMON_ERROR, e.getMessage());
             }
-        } else {
+        } else if (properties.containsKey(PropertyAnalyzer.PROPERTIES_FLAT_JSON_ENABLE)) {
```

Comment:
when enable is false, do we need to check other properties or return errors if they are set?